packages/core/src/extensions-shared/suggestion/SuggestionPlugin.ts

```diff
@@ -215,11 +211,7 @@ export const setupSuggestionsMenu = <
               triggerCharacter:
                 transaction.getMeta(pluginKey)?.triggerCharacter || "",
               queryStartPos: newState.selection.from,
-              items: items(""),
-              keyboardHoveredItemIndex: 0,
-              // TODO: Maybe should be 1 if the menu has no possible items? Probably redundant since a menu with no items
-              //  is useless in practice.
-              notFoundCount: 0,
+              items: getItems(""),
```

Comment:
- what happens with notFoundCount?
- can we cancel promises?
- how do we show loading state?